src/MonologHandler.php

```diff
@@ -59,7 +59,8 @@ protected function addContext(array $context = [])
         }
 
         // Add session data.
-        if ($session = $this->app->session->all()) {
+        $session = isset($this->app) ? $this->app->session->all() : null;
```

Comment:
Should'nt you check for `$this->app->has('session')` here instead of `isset($this->app)`?

Reply:
No, you shouldn't because if `$this->app` is `null` you cannot call the method `has`, and that's why it's failing now and it's better to check first with an `isset()`

Reply:
In my use cases it was only failing because `session` does not exist on `$this->app`, not because `$this->app` does not exist at all. In which situation `$this->app` is not defined?

Reply:
I've tested again and `$this->app` **does exists** in Lumen, that is the `->session` part that is failing.

Changing the `if` condition to `if ($this->app->has('session') && $session = $this->app->session->all()) {` solves the problem.